collectors/python.d.plugin/sslcheck/sslcheck.conf

```diff
@@ -0,0 +1,72 @@
+#
+# There are 2 sections:
+#  - global variables
+#  - one or more JOBS
+#
+# JOBS allow you to collect values from multiple sources.
+# Each source will have its own set of charts.
+#
+# JOB parameters have to be indented (using spaces only, example below).
+
+# ----------------------------------------------------------------------
+# Global Variables
+# These variables set the defaults for all JOBs, however each JOB
+# may define its own, overriding the defaults.
+
+# update_every sets the default data collection frequency.
+# If unset, the python.d.plugin default is used.
+# update_every: 1
+
+# priority controls the order of charts at the netdata dashboard.
+# Lower numbers move the charts towards the top of the page.
+# If unset, the default for python.d.plugin is used.
+# priority: 60000
+
+# penalty indicates whether to apply penalty to update_every in case of failures.
+# Penalty will increase every 5 failed updates in a row. Maximum penalty is 10 minutes.
+# penalty: yes
+
+# chart_cleanup sets the default chart cleanup interval in iterations.
+# A chart is marked as obsolete if it has not been updated
+# 'chart_cleanup' iterations in a row.
+# They will be hidden immediately (not offered to dashboard viewer,
+# streamed upstream and archived to backends) and deleted one hour
+# later (configurable from netdata.conf).
+# -- For this plugin, cleanup MUST be disabled, otherwise we lose latency chart
+chart_cleanup: 0
+
+# Autodetection and retries do not work for this plugin
+
+# ----------------------------------------------------------------------
+# JOBS (data collection sources)
+#
+# The default JOBS share the same *name*. JOBS with the same name
+# are mutually exclusive. Only one of them will be allowed running at
+# any time. This allows autodetection to try several alternatives and
+# pick the one that works.
+#
+# Any number of jobs is supported.
+#
+# -------------------------------
+# ATTENTION: Any valid configuration will be accepted, even if initial connection fails!
+# -------------------------------
+#
+# There is intentionally no default config for 'localhost'
+
+job_name:
+  name: myname                 # [optional] the JOB's name as it will appear at the
+                               # dashboard (by default is the job_name)
+  host: 'my-netdata.io'        # [required] the remote host address in either IPv4, IPv6 or as DNS name
+  port: 443                    # [required] the port number to check. Specify an integer, not service name
+  daysuntilexpiration: 5       # [required] days before the status is critical?
+  timeout: 10                  # [optional] the socket timeout when connecting
+  update_every: 3600           # [optional] the JOB's data collection frequency. As the chart is in days
+                               # remaining, a high value should be chosen
+  priority: 60000              # [optional] the JOB's order on the dashboard
+  penalty: yes                 # [optional] the JOB's penalty
+
```

Comment:
i suggest to copy config from another module, this config contains too much portcheck specific things